tests/test_snapCombine.py

```diff
@@ -15,158 +16,97 @@
-
-    @param[in] width image width (pixels)
-    @param[in] height image height (pixels)
-    @param[in] imMean mean of image plane
-    @param[in] varMean mean of variance plane
-    @param[in] maxMask maximum mask value; values will be uniformly chosen in [0, maxMask]
-    """
-    exp = afwImage.ExposureF(width, height)
-    exp.image.array[:, :] = np.random.poisson(imMean, size=exp.image.array.shape)
-    exp.variance.array[:, :] = np.random.poisson(varMean, size=exp.variance.array.shape)
-    exp.mask.array[:, :] = np.random.randint(0, maxMask + 1, size=exp.mask.array.shape)
-
-    return exp
-
 
-def simpleAdd(exp0, exp1, badPixelMask):
+def simple_add(exp0, exp1, bad_pixel_mask):
     """Add two exposures, avoiding bad pixels
     """
-    expRes = exp0.Factory(exp0, True)
-    weightMap = afwImage.ImageF(exp0.getDimensions())
+    result = exp0.Factory(exp0, True)
+    weights = afwImage.ImageF(exp0.getDimensions())
 
-    good0 = np.bitwise_and(exp0.mask.array, badPixelMask) == 0
-    good1 = np.bitwise_and(exp1.mask.array, badPixelMask) == 0
+    good0 = np.bitwise_and(exp0.mask.array, bad_pixel_mask) == 0
+    good1 = np.bitwise_and(exp1.mask.array, bad_pixel_mask) == 0
 
-    expRes.image.array[:, :] = np.where(good0, exp0.image.array, 0) + np.where(good1, exp1.image.array, 0)
-    expRes.variance.array[:, :] = np.where(good0, exp0.variance.array, 0) + \
+    result.image.array[:, :] = np.where(good0, exp0.image.array, 0) + np.where(good1, exp1.image.array, 0)
+    result.variance.array[:, :] = np.where(good0, exp0.variance.array, 0) + \
         np.where(good1, exp1.variance.array, 0)
-    expRes.mask.array[:, :] = np.bitwise_or(np.where(good0, exp0.mask.array, 0),
+    result.mask.array[:, :] = np.bitwise_or(np.where(good0, exp0.mask.array, 0),
                                             np.where(good1, exp1.mask.array, 0))
-    weightMap.array[:, :] = np.where(good0, 1, 0) + np.where(good1, 1, 0)
+    weights.array[:, :] = np.where(good0, 1, 0) + np.where(good1, 1, 0)
 
-    expRes.maskedImage /= weightMap
-    expRes.maskedImage *= 2  # want addition, not mean, where both pixels are valid
+    result.maskedImage /= weights
+    result.maskedImage *= 2  # want addition, not mean, where both pixels are valid
 
-    setCoaddEdgeBits(expRes.mask, weightMap)
+    setCoaddEdgeBits(result.mask, weights)
 
-    return expRes
+    return result
 
 
 class SnapCombineTestCase(lsst.utils.tests.TestCase):
-
     """A test case for SnapCombineTask."""
+    def setUp(self):
+        # Different x/y dimensions so they're easy to distinguish in a plot,
+        # and non-zero minimum, to help catch xy0 errors.
+        bbox = lsst.geom.Box2I(lsst.geom.Point2I(5, 4), lsst.geom.Point2I(205, 184))
+
+        center = lsst.geom.Point2D(50, 60)
+        # Two TestDatasets, as that is where VisitInfo is created.
+        dataset = lsst.meas.base.tests.TestDataset(bbox, visitId=1234, mjd=50000.0)
```

Comment:
Yeah, I don't need to override the mjd here at all, good point. I've removed that kwarg entirely here.

I did change so that the two ids are different (as they will be for two input exposures).